try:

    a = int(input('enter a num'))
    print(a+3)


except Exception as e:
    print('error',e)
